File: scrimage-filters/src/main/java/com/sksamuel/scrimage/filter/NashvilleFilter.java
package com.sksamuel.scrimage.filter;

import com.sksamuel.scrimage.PipelineFilter;
import com.sksamuel.scrimage.RGBColor;

import java.util.Arrays;

public class NashvilleFilter extends PipelineFilter {

    public NashvilleFilter() {
        super(Arrays.asList(
                new BackgroundBlendFilter(),
                new HSBFilter(0, -0.2f, 0.5f),
                new GammaFilter(1.2),
                new ContrastFilter(1.6),
                new VignetteFilter(0.9f, 1, 0.6f, new RGBColor(255, 140, 0, 255).toAWT())
        ));
    }
}

File: scrimage-filters/src/main/scala/com/sksamuel/scrimage/filter/NashvilleFilter.scala
package com.sksamuel.scrimage.filter

import java.awt.Color
import java.util

import com.sksamuel.scrimage.PipelineFilter

object NashvilleFilter extends PipelineFilter(util.Arrays.asList(
  BackgroundBlendFilter,
  HSBFilter(0, -0.2, 0.5),
  GammaFilter(1.2),
  ContrastFilter(1.6),
  VignetteFilter(0.9, 1, 0.6, new Color(255, 140, 0))
))

